opentelemetry-sdk/src/opentelemetry/sdk/_metrics/__init__.py

```diff
@@ -198,21 +198,23 @@ def shutdown(self):
             _logger.warning("shutdown can only be called once")
             return False
 
-        result = True
+        overall_result = True
```

Comment:
Doesn't have to be in this PR, but it might be useful to return a constant that can be checked against where shutdown is called

Reply:
The spec says for both [`MeterProvider`](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#shutdown) and [`MetricReader`](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#shutdown-1) `shutdown` methods: Shutdown SHOULD provide a way to let the caller know whether it succeeded, failed or timed out.

Since there are 3 possible outcomes, the 2 possible boolean results are not enought and we need something else.